### PR TITLE
Update SetFloatingHelpTextStyle.md

### PR DESCRIPTION
**Pull request by promraungdet**

### CODE REVIEW

File: HUD/SetFloatingHelpTextStyle.md [hunk 1]
```diff
@@ -6,14 +6,17 @@ aliases: ["0x788E7FD431BD67F1"]
 
 ```c
 // 0x788E7FD431BD67F1 0x97852A82
-void SET_FLOATING_HELP_TEXT_STYLE(Any p0, Any p1, Any p2, Any p3, Any p4, Any p5);
+void SET_FLOATING_HELP_TEXT_STYLE(Any p0, Any p1, hudColorIndex --[[ integer ]], Any p3, placement --[[ integer ]], ArrowPosition --[[ integer ]]);
```
Comment (blattersturm) [inline HUD/SetFloatingHelpTextStyle.md:9]: Incorrect syntax. See the code block: C. Not some weird Lua.

File: HUD/SetFloatingHelpTextStyle.md [hunk 2]
```diff
@@ -6,14 +6,17 @@ aliases: ["0x788E7FD431BD67F1"]
...
 ```
 
-## Parameters
```
Comment (blattersturm) [inline HUD/SetFloatingHelpTextStyle.md:12]: Why remove the entire Parameters section?

File: HUD/SetFloatingHelpTextStyle.md [hunk 3]
```diff
@@ -6,14 +6,17 @@ aliases: ["0x788E7FD431BD67F1"]
...
-* **p1**: 
-* **p2**: 
-* **p3**: 
-* **p4**: 
-* **p5**: 
+```
+hudColorIndex pastebin.com/d9aHPbXN  
+placement 1 = bottom, 2 = right, 3 = top, 4 = left
 
+Example
```
Comment (blattersturm) [inline HUD/SetFloatingHelpTextStyle.md:16]: ...?

**Review by blattersturm — CHANGES_REQUESTED** [review]
Please use the correct syntax. This would fail the build!

**Comment by technetium-cfx** [comment]
No updates; closing.

**Pull request closed without merging**